packages/hosted-form-v2/src/iframe-content/hosted-input-manual-order-payment-handler.ts

```diff
@@ -56,7 +55,6 @@ export default class HostedInputManualOrderPaymentHandler {
             const isError = get(response.body, 'type') === 'error';
 
             const isSuccessfulOfflineOrder =
-                isOfflinePaymentMethodId(data.paymentMethodId) &&
```

Comment:
I think this might cause regression with offline payment methods such as Cash On Delivery, Cheque, etc. 
Have you tested your PR against those payment methods?

Reply:
I will plan to test everything deeper after some feedback from the checkout folks.

Reply:
I think we might need to have a separate method for handling offline payment methods for payment providers.